src/main/resources/templates/repo/main-repo-shared-resources-template.json.vpt

```diff
@@ -494,6 +494,50 @@
 				]
 			}
 		},
+		"${stack}${instance}SynapseTempCredentialsServiceRole": {
+			"Type": "AWS::IAM::Role",
+			"Properties": {
+				"AssumeRolePolicyDocument": {
+					"Version": "2012-10-17",
+					"Statement": [
+						{
+							"Effect": "Allow",
+							"Principal": {
+								"AWS": {"Ref": "AWS::AccountId"}
```

Comment:
Velocity switch statement, in develop, account ID. In prod, refer to the SynapseWorker role.